Updating docs for installation scripts

dimpar opened the pull request:
Refs https://github.com/keep-network/keep-core/issues/3135

This doc explains how to run installation scripts: `install.sh, initialize.sh, start.sh` and what each script does under the hood.

nkuba commented on docs/development/README.adoc:
````diff
@@ -35,22 +87,83 @@ The `+initialize.sh+` script will:
+```
 
-* run `./scripts/install.sh`. You need to run it only once.
+Afer executing `install.sh` and `initialize.sh` now you can run `./scripts/start.sh` to start a client. It will prompt you to chose from the available config files (that should match the one chosen for `./scripts/initialize.sh`) and the log level. 
````
```suggestion
=== start.sh

Afer executing `install.sh` and `initialize.sh` now you can run `./scripts/start.sh` to start a client. It will prompt you to chose from the available config files (that should match the one chosen for `./scripts/initialize.sh`) and the log level. 
```

nkuba commented on docs/development/README.adoc:
````diff
@@ -35,22 +87,83 @@ The `+initialize.sh+` script will:
+Please refer to
+`./scripts/start.sh --help` for more info
+
+```
+./scripts/start.sh --help
+
+Usage: ENV_VAR(S) ./scripts/start.sh --config-dir <path-to-configuration-files>
+
+Environment variables:
+
+        KEEP_ETHEREUM_PASSWORD: Ethereum account password. Required only for 'local' network. Default value is 'password'
+
+Command line arguments:
+
+        --config-dir: Path to a client configuration files
+```
````
Let's remove the help output.

nkuba commented on docs/development/README.adoc:
```diff
@@ -7,26 +7,78 @@
+configuring T peers. +
 
-== Quick installation
+Additionaly, to start a client we need to provide config options either by creating a config file or passing the required parameters. For simple installation using a config file please refer to a <<sample_config_file,sample configuration>> file that is placed under `/configs/` dir.
```
We should add a paragraph that it is required to create configuration files for each instance one wants to run and place them inside the `configs/` directory.

nkuba commented:
@dimpar could you also please merge the main branch? It has the docs updated and a job that will post a preview link for docs.

dimpar commented:
@nkuba I've addressed all of your comments in a single commit cause some parts were related to each other and it was hard to separate them. [dce7cb6](https://github.com/keep-network/keep-core/pull/3170/commits/dce7cb6001a47893af190662b4c865f19173d0ad) 

Regarding removing `--help` output, I'm actually 50/50 on this one. I see that a user can type `--help` himself, but on the other hand, it also has some value just displaying the available options so a user has a quick overview while reading the docs. Also, I believe @pdyraga suggested displaying the `--help` output as well.

nkuba commented:
> Regarding removing `--help` output, I'm actually 50/50 on this one. I see that a user can type `--help` himself, but on the other hand, it also has some value just displaying the available options so a user has a quick overview while reading the docs. Also, I believe @pdyraga suggested displaying the `--help` output as well.

A disadvantage of hardcoded command output is that it will most likely be outdated soon and we won't remember to update the doc once the command changes 🤷🏻
